SinaiStCatherine/SinaiEt006.xml

```diff
@@ -140,6 +140,10 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                       <locus from="111v" to="151v"/>
                       <title type="complete" ref="LIT1961Mashaf">A version of Horologion</title>
                       <textLang mainLang="gez"/>
+                      
+                      <note>On fol. 111v is an exerpt from Mawase'et and should be treated separately, cp. 
```

Comment:
This note also looks more like a comment. I would suggest to either turn it into a comment to be edited in the future or update its transcription and encoding

Reply:
encoded.